public/img/Vector 3.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="15" height="19" viewBox="0 0 15 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0.414978 18L13.4099 8.55556L0.414978 1" stroke="white"/>
+</svg>
```

Comment:
It would be better if you gave some more descriptive names to this svgs so it's more obvious what they do, ex. "angle-left", "angle-right". Or maybe you could use "<<" and ">>" in text if it looks good?
I like that the angles point right when you add an item and point left when you go back home, it gives a nice sense of continuity :slightly_smiling_face:

Reply:
Agree with that sense of continuity!! A surprisingly delightful user experience!

Reply:
Ok thanks. I added IDs to identify them👍🏻👍🏻